lisa/features/infiniband.py

```diff
@@ -26,3 +52,63 @@ def is_over_sriov(self) -> bool:
     # example SKU: Standard_H16mr
     def is_over_nd(self) -> bool:
         raise NotImplementedError
+
+    def get_ib_interfaces(self) -> List[Tuple[str, str, str]]:
```

Comment:
If those methods are the same to all platform, they should be put in the general feature implementation.